pandas/_libs/util.pxd

```diff
@@ -1,7 +1,7 @@
 cimport numpy as cnp
 from numpy cimport ndarray
 
-from pandas._libs.tslibs.util cimport *
+from .tslibs.util cimport *
```

Comment:
why are you changing this? we generally use absolute imports

Reply:
Oops. This slipped in while I was rebasing the patch for upstream. Removed.